### DISCUSSION TITLE
Won Auction Verification

### DISCUSSION THREAD

**Discussion by cdt-eth** [nounsDAO/nouns-monorepo#396] [Frontend] [answered]
When I win an auction, I get the “verify” button on the Noun I won. If I click that it gives me 3 fields: signature address, signature, and message. What’re the purpose of these fields? If someone wins a Noun in real life, will they have to tap on that and provide the info if they want to settle the current auction?

**Comment by solimander** [comment, marked as answer]
The /sign and /verify routes render the same page which gives the visitor the ability to sign any message with their account. The lower section was where we'd plug the signed message payload to verify the signature was cryptographically correct. That was an early way to make sure that discord users were actual Noun holders. You don't have to verify in order to settle an auction.
